src/Grid/Grid.js

```diff
@@ -81,9 +81,7 @@ class Grid extends React.Component {
           />
           { body }
           {
-            footerRenderer &&
-              aggregate &&
-                footerRenderer(aggregate)
+            aggregate && footerRenderer(aggregate)
```

Comment:
Hopefully footerRenderer will be defined... again, the aggregate check is what you need to remove